doc/adding-new-regression-tests.md

```diff
@@ -0,0 +1,71 @@
+# Adding new regression tests
+
+1. Update the flow to replace any single line conditionals around `Phraselist`s with multiple line conditionals. This is so that we get useful information from the running the coverage utility. Single line conditionals will show up as having been exercised irrespective of whether they caused something to be added to the `Phraselist`.
+
```

Comment:
Thanks. Yes, I think you're right that they have (almost?) entirely disappeared from question blocks. We already have a Trello card to remove documentation associated with them: https://trello.com/c/zODxvA3Y/78-remove-documentation-references-to-phraselists